src/index.js

```diff
@@ -1,35 +1,48 @@
+// import _ from 'lodash';
+// import { add } from 'lodash';
```

Comment:
- [ ] [OPTIONAL] Great job 👍 . Kindly make sure you remove all commented code. This would make your code neater and easier to read. It is also be practise 🙏

Reply:
Sure, i will delete this @Ugboaja-Uchechi